src/sagas/Captcha.js

```diff
@@ -21,8 +23,7 @@ export function* sendCaptcha() {
         csrf_token: state.config.csrf_token,
         tou_accepted: state.email.tou_accepted
       };
-      
-    //await sleep(10000);
+    yield delay(10000);
```

Comment:
I’m wondering why we choose to add a delay to the saga? I ask because it adds a 10 second wait where the website looks unresponsive even on a single click. Multiple clicks reset the start of the 10 second wait. I am not sure the wait is very helpful for our users. 

I expected the solution to be disabling of the button while the api call is made, is there any reason why we are not doing that?

Reply:
I thought this issue occurred du to the commented out code, so I reinstated the code since I thought it was quick and right solution. I do agreed with you, 10 second is long time to wait for our users. so I have now removed the delay function and instead added `disabled `button when user clicking done until it gets a response from backend.  if user get failed "captcha" the button will be available again, if success user get information(successfully registered).